dev-libs/yazpp/yazpp-1.8.1.recipe

```diff
@@ -0,0 +1,159 @@
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libyaz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	yazpp${secondaryArchSuffix}_devel = $portVersionCompat
+	cmd:yazpp_config$commandSuffix
+	devel:libyazpp$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	yazpp$secondaryArchSuffix == $portVersion base
+	devel:libyaz$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	yazpp${secondaryArchSuffix}_tools = $portVersionCompat
+	cmd:yazpp_illclient$commandSuffix
+	cmd:yazpp_ztest$commandSuffix
+	"
+
+REQUIRES_tools="
+	yazpp$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	lib:libyaz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libyaz$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage yazpp$secondaryArchSuffix \
+	$libDir/libyazpp.so.$libVersion
+
+PATCH()
+{
+	# Fix library paths for Haiku
+	sed -i "s|/lib\"|/$libDir\"|g" configure.ac
+}
+
+BUILD()
+{
+	export LDFLAGS="-lnetwork"
+	
+	# Create symlink for libxml headers that YAZ expects
+	mkdir -p /tmp/include
+	ln -sf $portPackageLinksDir/devel~libxml2$secondaryArchSuffix/develop/headers/libxml2/libxml /tmp/include/libxml
+	export CPPFLAGS="-I/tmp/include"
+	
+	autoreconf -fi
+	
+	runConfigure ./configure \
+		--prefix=$prefix \
+		--bindir=$commandBinDir \
+		--libdir=$libDir \
+		--includedir=$includeDir \
+		--datadir=$dataDir \
+		--mandir=$manDir \
+		--disable-static
+	
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	
+	# Clean up libtool files
+	rm -f $libDir/*.la
+	
+	# Prepare development libraries
+	prepareInstalledDevelLibs libyazpp libzoompp
+	fixPkgconfig
+	
+	# Package development components
+	packageEntries devel \
+		$developDir \
+		$commandBinDir/yazpp-config \
+		$dataDir/aclocal
+	
+	# Only package tools that actually exist
+	# Check what executables were built in the source tree
+	if [ -f $commandBinDir/yazpp-illclient ]; then
+		packageEntries tools $commandBinDir/yazpp-illclient
+	fi
+	if [ -f $commandBinDir/yazpp-ztest ]; then
+		packageEntries tools $commandBinDir/yazpp-ztest
+	fi
+	
+	# Move man pages if they exist  
+	if [ -d $manDir ]; then
+		if [ -f $manDir/man1/yazpp-config.1 ]; then
+			packageEntries devel $manDir/man1/yazpp-config.1
+		fi
+		if [ -f $manDir/man1/yazpp-illclient.1 ]; then
+			packageEntries tools $manDir/man1/yazpp-illclient.1
+		fi
+		if [ -f $manDir/man1/yazpp-ztest.1 ]; then
+			packageEntries tools $manDir/man1/yazpp-ztest.1
+		fi
+	fi
```

Comment:
Do you expect the package contents to change randomly, or why are these if statements needed?